open-craft/opencraft
# BB-834: Fix simple-theme deployments

This PR fixes a lot of styling errors and improves readability for simple-theme deployments on Ocim.

**Testing instructions:**
Note: These changes are deployed to Ocim stage.
1. Use Ocim stage to deploy an instance using [the beta testing application form](https://stage.console.opencraft.com/registration/) and this [configuration branch](https://github.com/open-craft/configuration/pull/87).
2. Check if the theme is correctly deployed in the appserver.

I already done this with 2 sets of colors, so you can check the result at:
Light colors: https://stage.console.opencraft.com/instance/3230/edx-appserver/1022/
Darker colors: https://stage.console.opencraft.com/instance/3230/edx-appserver/1020/

**Reviewers:**
